INSTALL.rst

```diff
@@ -60,87 +102,150 @@ Also the following ports are exposed on the Docker host:
 - ``9300``: Elasticsearch
 - ``15672``: RabbitMQ management console
 
-**Dependencies**
-
-Zenodo depends on PostgreSQL, Elasticsearch, Redis and RabbitMQ.
-
 
 Development installation
 ------------------------
-For core developers it is often faster and easier in the long run to not to use
-Docker, with a bit of extra up-front work to configure your box. Note, however
-that Docker provides the most similar environment to the production
-environment.
 
-First check out the source code:
+For the development setup we will reuse the Zenodo docker image from
+previous section to run only essential Zenodo services, and run the
+application code and the Celery worker outside docker - you will want to
+have easy access to the code and the virtual environment in which it will be
+installed.
+
+.. note::
+
+    Since docker will be mapping the services to the default system
+    ports on localhost, make sure you are not running PostgreSQL,
+    Redis, RabbitMQ or Elasticsearch on those ports in your system.
+
+Similarly to how we previously ran ``docker-compose up`` to run full-stack
+Zenodo, this time we run only four docker nodes with the database,
+Elasticsearch, Redis and RabbitMQ:
 
 .. code-block:: console
 
-    $ cd ~/src/
-    $ git clone https://github.com/zenodo/zenodo.git
-    $ cd zenodo
+    $ docker-compose up db es cache mq
 
-Next, create a virtual environment (using
-`virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/>`_):
+Keep the docker-compose session above alive and in a separate shell, create a
+new Python virtual environment using virtualenvwrapper
+(`virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/>`_),
+in which we will install Zenodo code and its dependencies:
 
 .. code-block:: console
 
-    $ mkvirtualenv zenodo
-    (zenodo)$
+    (zenodo)$ mkvirtualenv zenodo
```

Comment:
You get the `(zenodo)` prompt prefix after running the command, maybe visually indicate this like:

```
$ mkvirtualenv zenodo
(zenodo)$ 
```